File: Cpp/1008.cpp
#include <iostream>
#include <iomanip>
 
using namespace std;
 
int main() {
 
    int num, hor;
    double d, total;

    cin >> num;
    cin >> hor;
    cin >> d;

    total = hor * d;

    cout << "NUMBER = " << num << "\n";
    cout << fixed << setprecision(2);
    cout << "SALARY = U$ " << total << "\n";
    
    return 0;
}

File: Haskell/1008.hs
import Text.Printf ( printf )

main ::  IO ()
main = do
    number <- readLn :: IO Int
    printf "NUMBER = %i\n" number
    horas <- readLn :: IO Double
    valor_por_hora <- readLn :: IO Double
    let salary = horas * valor_por_hora
    printf "SALARY = U$ %.2f\n" salary